tests/__init__.py

```diff
@@ -34,6 +34,17 @@ def __deepcopy__(self, memo: Any) -> Any:
         raise NotImplementedError()
 
 
+class Dataframe:
+    def __init__(self) -> None:
+        pass
+
+    def __eq__(self, other: Any) -> Any:
+        return self
+
+    def __bool__(self) -> None:
+        raise ValueError
```

Comment:
add a comment explaining what this achieves.

Reply:
let me know if the comments I added here work